scripts/exgdas_global_marine_analysis_run.sh

```diff
@@ -108,6 +132,17 @@ concatenate_bump 'bump3d'
 clean_yaml var.yaml
 $APRUN_SOCAANAL $JEDI_BIN/soca_var.x var.yaml 2>var.err
 
+
+# Prepare increment for MOM6 iau
+if [[ "$DA_OCN_IAU" =~ [yYtT1] ]]; then
```

Comment:
no need for this, we will always use IAU.

Reply:
Ok, it is updated